test/addons-napi/test_bigint/test_bigint.c

```diff
@@ -86,11 +86,26 @@ static napi_value TestWords(napi_env env, napi_callback_info info) {
   return output;
 }
 
+// throws RangeError
+static napi_value CreateTooBigBigInt(napi_env env, napi_callback_info info) {
+  int sign_bit = 0;
+  size_t word_count = SIZE_MAX;
+  uint64_t words[word_count];
```

Comment:
I’m not sure what the compiler makes out of this, but you can’t actually create a stack-allocated array of size `SIZE_MAX` … that’s most likely the cause of the segfault?